docs/source/monitoring/metrics.mdx

```diff
@@ -3,7 +3,7 @@ title: Metrics and logging
 description: How to monitor Apollo Server's performance
 ---
 
-Apollo Server integrates seamlessly with Apollo Studio to help you monitor the execution of your GraphQL operations. It also provides configurable mechanisms for logging each phase of a GraphQL operation.
+Apollo Server integrates seamlessly with [Apollo Studio](/studio) to help you monitor the execution of your GraphQL operations. Apollo Studio additionally provides configurable mechanisms for logging each phase of a GraphQL operation.
```

Comment:
I think the "it" here was referring to Apollo Server

Reply:
Ooops, too many "S" words 🤦‍♀️